domains/certificates/chart/values.yaml

```diff
@@ -12,3 +12,7 @@ postgres:
   size: 10Gi
 rabbitmq:
   replicaCount: 1
+metrics:
+  prometheus:
+    enabled: true
+    # serviceMonitorReleaseLabel: ""
```

Comment:
I would like to have `additionalLabels` where you can add both key and value for labels. It is much more open for configuration and less confusing about what a "release" label is.

Reply:
So which objects should these labels be added to? In certificates domain we have Deployment-objects, Service-objects, ConfigMap-objects, StatefulSet-objects, RabbitMQCluster-objects and ServiceMonitor-objects. Should the labels in `additionalLabels` be added to all of these objects?

Reply:
I have added `additionalLabels` in 24c11053364fbd104da0ea4ecc832a9d47000dad, but it only adds labels to the ServiceMonitor. Please let me know, if this is what you are asking for.